iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ThreeWindingsTransformerImpl.java

```diff
@@ -37,7 +34,7 @@
 
         private double ratedS;
 
-        private CurrentLimits limits;
+        private OperationalLimitsHolderImpl operationalLimitsHolder;
```

Comment:
Should be declared as final?

Reply:
It is only set in `setTransformer()` i.e. when the transformer is created which is only **after** the legs are created, which explains why it isn't final.